.github/actions/fullnode-sync/fullnode_sync.py

```diff
@@ -256,6 +347,13 @@ def setup_fullnode_config(bootstrapping_mode, continuous_syncing_mode, data_dir_
   # Enable storage sharding (AIP-97)
   fullnode_config['storage']['rocksdb_configs'] = {"enable_storage_sharding": True}
```

Comment:
Sharding is enabled by default now, could be removed.

Reply:
Done 😄

Reply:
Ok, good to know. We can revisit this later.